File: 프로그래머스/0/120835. 진료 순서 정하기/진료 순서 정하기.py
def solution(emergency):
    num = [i for i in range(len(emergency))]
    num.sort(key=lambda x: emergency[x], reverse=True)
    result = [0] * len(emergency)
    for i in range(len(emergency)):
        result[num[i]] = i + 1
    return result

File: 프로그래머스/0/120835. 진료 순서 정하기/진료 순서 정하기.js
function solution(emergency) {
    let num = Array.from({ length: emergency.length }, (_, index) => index);
    num.sort((a, b) => emergency[b] - emergency[a]);
    let result = new Array(emergency.length);
    for (let i = 0; i < emergency.length; i++) {
        result[num[i]] = i + 1;
    }
    return result;
}
